### PR TITLE
Add cloudwatch slack notifier

### PR DESCRIPTION
**Pull request by mweiden**

### CODE REVIEW

File: infrastructure.tf [hunk 1]
```diff
@@ -422,3 +425,116 @@ resource "aws_iam_role_policy" "cwl-firehose-staging" {
...
+            "Resource": "arn:aws:logs:us-east-1:861229788715:*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "logs:CreateLogStream",
+                "logs:PutLogEvents"
+            ],
+            "Resource": [
+                "arn:aws:logs:us-east-1:861229788715:log-group:/aws/lambda/cloudwatch-slack-notifications:*"
```
Comment (parthshahva) [inline infrastructure.tf]: shouldn't the account id be interpolated here rather than checked in?

Comment (mweiden) [reply infrastructure.tf]: darn, true

**Comment by mweiden** [comment]
@parthshahva I forgot to make the deployment change. I will make that now. Re. the separation of the files, this is my plan for the coming `sicc` PR. So the focus of this PR is to get the slack notifier out. The focus of that PR will be to get everything modular.

File: Makefile [hunk 2]
```diff
@@ -12,17 +13,25 @@ test:
 	$(MAKE) -C exporters/gcp_to_cwl/ test
 	$(MAKE) -C es_managers/es_idx_manager/ test
 
+.PHONY: plan-infrastructure
+plan-infrastructure:
+	./infrastructure.sh apply
```
Comment (parthshahva) [inline Makefile]: This should be a plan instead of an apply right?

Comment (mweiden) [reply Makefile]: Oh true. This is garbo. Removing it.

**Pull request merged**